File: Nogle/count_prime.java
import java.util.*;

public class count_prime {
	
	public int count(int n) {
		boolean[] tmp = new boolean[n];
		int count = 0;
		for(int i = 2; i < n; i++) {
			if(tmp[i] == false) {
				count++;
				
				for(int j = 2; i*j < n; j++){
					tmp[i*j] = true;
				}
			}
		}
		return count;
	}
	
	public static void main(String[] args) {
		
		int n = 14;
		count_prime cp = new count_prime();
		System.out.print("number is : " + cp.count(n)+ "\n");
	}
	
}

File: Nogle/count_prime.py
import os   

def count_prime(number):
    list = [0] * number
    ans = 0
    for i in range(2,number):
	    if list[i] == False:
		    ans += 1
		    for j in range(2,number):
			    if j*i >= number:
				    break
			    else:
				    list[j*i] = True
	
    return ans
	
def main():
    n = 14
    print(count_prime(n))
	
if __name__ == '__main__':
    main()